src/Common/src/Interop/Windows/user32/Interop.GetWindowTextLengthW.cs

```diff
@@ -9,7 +9,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32, EntryPoint = "GetWindowTextLengthW")]
-        public static extern int GetWindowTextLength(IntPtr hWnd);
+        [DllImport(Libraries.User32, SetLastError = true, ExactSpelling = true)]
+        public static extern int GetWindowTextLengthW(IntPtr hWnd);
```

Comment:
If it's `GetWindowTextLength` doesn't pinvoke infra find `GetWindowsTextLengthW`, even without passing EntryPoint? I thought it automatically tried suffixing W.
if so, missing a W is nice - easier to read and normalized for when messing with API baselines and such.